### PR TITLE
Bump admin UI

### PR DESCRIPTION
**Pull request by mairatma**
#### What problem is this solving?

Updates the @vtex/admin-ui to the most recent version. 

#### How to test it?

Use [this workspace](https://mairab2badminui--b2bstoreqa.myvtex.com/admin/b2b-organizations/organizations) and navigate around the organization admin pages. The most affected one is the [Custom Fields page](https://mairab2badminui--b2bstoreqa.myvtex.com/admin/b2b-organizations/organizations/#/custom-fields) which uses the library more than the others and required a fix for a breaking change from the upgrade. I've used the buttons/dropdowns and it's all working like before.

### CODE REVIEW

File: react/admin/DefaultCustomField.tsx [hunk 1]
```diff
@@ -216,9 +216,9 @@ const DefaultCustomField: React.FC<CustomFieldProps> = ({
                     )}
                     variant="secondary"
                     onClick={() => handleDeleteDropdownItem(i)}
-                    csx={{
+                    className={csx({
```
Comment (mairatma) [inline react/admin/DefaultCustomField.tsx:219]: This was a breaking change from this version: the `csx` prop doesn't exist anymore, you need to send it within `className` instead, as explained [here](https://admin-ui.vercel.app/releases).

**Comment by mairatma** [comment]
> Overall looks good, but when I go to the "Organization Requests" tab, it breaks: https://mairab2badminui--b2bstoreqa.myvtex.com/admin/b2b-organizations/organizations/#/requests

You're right, but I can't seem to open [that page in master](https://b2bstoreqa.myvtex.com/admin/b2b-organizations/organizations/#/requests) either 🤔 . Does it open for you? The console seems to show the same error. We haven't received reports for this though, so it may be something with this particular account, some bad data that's fataling... I'll investigate better, it seems like we need a fix regardless.

**Comment by mairatma** [comment]
Just FYI: in master we don't render the stack traces of the errors, so the page just shows blank, but we log the error in the console instead. I believe it's the same one.

**Comment by enzomerca** [comment]
I see! So it is not an issue with your change... but can we test it in a working test store then? Just to make sure we are not breaking anything on that page/tab. https://b2bsuite.myvtex.com/admin/b2b-organizations/organizations/#/requests should be working

**Comment by mairatma** [comment]
@enzomerca I've confirmed that it was some kind of bad data in this account. There was one organization request without an admin user and without a cost center (both of which are required). I first thought about changing the frontend to pretend these fields were optional, but that makes no sense. Instead, I've deleted this bad organization request. Let's keep an eye open in case a real production account comes up with a similar issue though.

**TLDR: I've confirmed that this was not related to this PR, it was a bad data issue with a specific organization request for this test account.**

**Review by enzomerca — DISMISSED** [review]

**Comment by mairatma** [comment]
FYI: I'm struggling with this sonar run, I believe it's the expired token taht we've had in other repos, so [I've asked for an update from the DX team](https://vtex.slack.com/archives/C03HWBE913Q/p1689010893795429).

**Comment by mairatma** [comment]
@enzomerca SonarCloud is now passing, but your approval disappeared from the PR 🤦‍♀️ 😩 . Can you take another look when you have some time?

**Review by enzomerca — APPROVED** [review]
